### PR TITLE
Vlsav configure cluster cookie

### PR DESCRIPTION
**Pull request by vlsav**
for https://github.com/tarantool/tarantool-operator/issues/112

### CODE REVIEW

File: examples/kv/helm-chart/values.yaml [hunk 1]
```diff
@@ -3,6 +3,7 @@
 ClusterEnv: dev
 ClusterName: examples-kv-cluster
 ClusterDomainName: cluster.local
+ClusterCookie: "tnt-kv-cookie"
```
Comment (vanyarock01) [inline examples/kv/helm-chart/values.yaml:6]: Tarantool Cartridge sets the default value of the cookie itself.
https://www.tarantool.io/ru/doc/latest/book/cartridge/cartridge_api/modules/cartridge/#cfg-opts-box-opts

Let's not change it unnecessarily.

Comment (chelsEg) [reply examples/kv/helm-chart/values.yaml:6]: Agree with @vlsav it's more flexible.
For example, in my case I have docker image and use him in different projects, but have one password for all, think it's not secure.

Comment (vanyarock01) [reply examples/kv/helm-chart/values.yaml:6]: I mean, you shouldn't add this param to the default chart `values.yaml`.
in other words, you need to leave only what you fixed in `deployment.yaml`.

Comment (vlsav) [reply examples/kv/helm-chart/values.yaml]: adjected pull request according to feedback (in values.yaml line commented, with comment this is an example how to override cluster cookie definition)

**Comment by kluevandrew** [comment]
Rejected because helm charts would be placed in standalone repo in near future.

**Pull request closed without merging**